src/LiveComponent/src/Attribute/LiveProp.php

```diff
@@ -58,6 +58,13 @@ final class LiveProp
      */
     private null|string|array $onUpdated;
 
+    /**
+     * @var bool
+     *
+     * Tells if this property should be bound to the URL
+     */
+    private bool $url;
```

Comment:
If i did not read the code i'd believe you expected a URL string. Maybe something more explicit like liveParam ou boundToUrl ?

Reply:
Seems legit. I also consider future url options that could be passed to `LiveProp` constructor. I see two ways: 

Allow an object to configure options:

Then `urlMapping` will be better imo. We could still allow a boolean, so `true` leads to basic defaults, and `false` disables the binding. Depending on when these new options get available, it could also improve BC.

Pass each option separately: 

Then `boundToUrl` seems good to me. But maybe passing each option separately will complexify the constructor too much.

wdyt?

Reply:
No particular preference... i wonder is something "query" would not be more accurate / self-explanatory ..

If it was a new attribute (may be an idea if we need many attributes) ...  i think i'd call it LiveQueryParam so maybe .. something like queryParam ? 
mapQuery ? 
queryName ? 

What do you have in mind for the object / multiple options ? That could help me better see the vision you have for this :)

Reply:
Potential new options are `alias`, `keep` and `history` (at least if we want to stick with the [Livewire feature](https://livewire.laravel.com/docs/url)). So I thought about either a `QueryMapping` object:
```php
class QueryMapping {
    public function __construct(string $alias, bool $keep = false, bool $history = false) { ... }
}
```

Or params directly in the `LiveProp` constructor: `queryAlias`, `queryKeep`, etc. (or `urlAlias`...).

Even if these features are out of this PR scope, I think that keeping them in mind to choose the appropriate naming may worth it . But maybe I'm overthinking this ^^

Reply:
What would happen if two LiveProps had different values for keep or history ?

Reply:
For the `keep` option, it'd actually be used only on first load to tell if the param should appear in the URL, even if it's not specified. So each prop handle its own parameter, they work independently.

For the `history` option, we challenged this in [this comment](https://github.com/symfony/ux/pull/1230#issuecomment-1796349698). Basically, we'd add a new history entry if there is at least one param involved in the URL modification that has the option set.

Reply:
Ok that seems logical, and i don't see how we could do "smarter" if a developper uses contradictory values.

👍